doc/modules/multiclass.rst

```diff
@@ -486,7 +462,37 @@ Note that all classifiers handling multiclass-multioutput (also known as
 multitask classification) tasks, support the multilabel classification task
 as a special case. Multitask classification is similar to the multioutput
 classification task with different model formulations. For more information,
-see the relevant estimator documentation.
+see the relevant estimator documentat
+
+Below is an example of multiclass-multioutput classification:
+
+    >>> from sklearn.datasets import make_classification
+    >>> from sklearn.multioutput import MultiOutputClassifier
+    >>> from sklearn.ensemble import RandomForestClassifier
+    >>> from sklearn.utils import shuffle
+    >>> import numpy as np
+    >>> X, y1 = make_classification(n_samples=10, n_features=100,
+    ...                             n_informative=30, n_classes=3,
+    ...                             random_state=1)
+    >>> y2 = shuffle(y1, random_state=1)
+    >>> y3 = shuffle(y1, random_state=2)
+    >>> Y = np.vstack((y1, y2, y3)).T
+    >>> n_samples, n_features = X.shape # 10,100
+    >>> n_outputs = Y.shape[1] # 3
+    >>> n_classes = 3
+    >>> forest = RandomForestClassifier(random_state=1)
+    >>> multi_target_forest = MultiOutputClassifier(forest, n_jobs=-1)
```

Comment:
While we are at it, let's limit the number of workers to avoid memory crash with starting to many Python subprocesses when running those docstests on machines with many cores.

```suggestion
    >>> multi_target_forest = MultiOutputClassifier(forest, n_jobs=2)
```